hack/app-sre/saas-objects.yaml

```diff
@@ -0,0 +1,6504 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
is it necessary to to store this duplicated data? also do we update this everytime we update the CRD resources?

Reply:
The duplication is no different in spirit to stuffing all the CRDs etc into the OLM bundle. (Or vendoring deps for that matter.)

> also do we update this everytime we update the CRD resources?

Yes, this file is generated and put in the `make verify` path for exactly that reason: so that the various manifests can live in one place, and we're forced to update automatically any time they change.

Reply:
would it better to keep the scripts in hack, but actual objects in a directory that doesn't say hack ?
when making a PR, this will create a diff in hack directory which can look odd.

also is it possible to atleast remove this objects list file? and just maybe create this as temp file and create the saas bundle from it, and then delete this?
one less diff when making changes to CRDs

Reply:
> would it better to keep the scripts in hack, but actual objects in a directory that doesn't say hack ?

FWIW, the [OLM template](https://github.com/openshift/hive/blob/master/hack/olm-registry/olm-artifacts-template.yaml) is under hack. Also, I really want to group everything app-sre into the same directory. (TBH, not sure why the initial dir needs to be named "hack" in the first place.)

> also is it possible to atleast remove this objects list file?

Yeah, I can do that.